### PR TITLE
Adds ConnectionInfo to Rpc callback

### PR DESCRIPTION
**Pull request by ThadHouse**
Only did the callback this time instead of the RpcCallInfo. I'll do that in a separate PR. This has been tested on both linux and windows to work properly

### CODE REVIEW

File: src/RpcServer.cpp [hunk 1]
```diff
@@ -73,8 +75,8 @@ bool RpcServer::PollRpc(bool blocking, double time_out, RpcCallInfo* call_info)
     } else {
 #if defined(_MSC_VER) && _MSC_VER < 1900
       auto timeout_time = std::chrono::steady_clock::now() + 
-          std::chrono::duration<int64_t, std::nano>(static_cast<int64_t>
-          (time_out * 1e9));
+           std::chrono::duration<int64_t, std::nano>(static_cast<int64_t>
+           (time_out * 1e9));
```
Comment (PeterJohnson) [inline src/RpcServer.cpp]: 5 spaces?

Comment (ThadHouse) [reply src/RpcServer.cpp]: Oh whoops. Fixed, and also rebased off of the newest master

File: src/Storage.cpp [hunk 2]
```diff
@@ -1392,13 +1404,19 @@ unsigned int Storage::CallRpc(StringRef name, StringRef params) {
     // gracefully anyway.
     auto rpc_callback = entry->rpc_callback;
     lock.unlock();
+    ConnectionInfo conn_info;
+    conn_info.remote_id = "Server";
+    conn_info.remote_ip = "LocalHost";
```
Comment (PeterJohnson) [inline src/Storage.cpp]: Usually this would be all lowercase "localhost" or "127.0.0.1"?

Comment (ThadHouse) [reply src/Storage.cpp]: Ah I didn't know. I'll fix that.

Fixed

**Review by PeterJohnson — APPROVED** [review]

**Pull request merged**